frontend/src/entry-points/shop.tsx

```diff
@@ -45,33 +45,32 @@ function AppInfoModal(item: AppInfoWithState, onDownload: () => void) {
   const [isExpanded, setIsExpanded] = createSignal(false)
 
   return (
-    <li class="w-full border rounded p-4 shadow">
+    <li class="w-full border rounded p-4 shadow" onClick={() => setIsExpanded(!isExpanded())}>
       <div class="flex items-center justify-between gap-2">
         <img src={`data:image/png;base64,${item.image!}`} alt={item.name} class="h-20 w-20 rounded-xl object-cover" />
         <div class="flex-grow-1 overflow-hidden">
           <h2 class="text-xl font-semibold">{item.name}</h2>
           <p class="max-width-text truncate text-gray-600">{item.description}</p>
         </div>
-        {item.state === AppState.Initial && <button class="justify-self-center btn" onClick={onDownload}> Add </button>}
+        {item.state === AppState.Initial && <button class="justify-self-center px-2 btn" onClick={onDownload}> Add </button>}
         {item.state === AppState.Downloading && <p class="unimportant"> Downloading.. </p>}
         {item.state === AppState.DownloadCancelled && <p class="text-red"> Download cancelled </p>}
         {item.state === AppState.Received && <p class="font-bold text-amber-400"> Received in Chat </p>}
       </div>
       {
         isExpanded() && (
           <>
-            <p class="my-2 text-gray-600">{item.description}</p>
-            <hr />
```

Comment:
personally I liked the horizontal line separating the app description from the other metadata fields, but not biggie, what I find a bit ugly is that the description is duplicated, IMHO when expanding to see all metadata, it would look nicer to hide the other description label shown in the collapsed mode

Reply:
@r10s also complained about it but I have already played around with a transition which sadly feels very clunky. If I change this, then in a seperate PR, as this needs some work.